apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -950,6 +950,150 @@ describe('LIVECHAT - contacts', () => {
 		});
 	});
 
+	describe('[GET] omnichannel/contacts.checkExistence', () => {
+		let contactId: string;
+		let association: ILivechatContactVisitorAssociation;
+
+		const email = faker.internet.email().toLowerCase();
+		const phone = faker.phone.number();
+
+		const contact = {
+			name: faker.person.fullName(),
+			emails: [email],
+			phones: [phone],
+			contactManager: agentUser?._id,
+		};
+
+		before(async () => {
+			await updatePermission('view-livechat-contact', ['admin']);
+			const { body } = await request
+				.post(api('omnichannel/contacts'))
+				.set(credentials)
+				.send({ ...contact });
+			contactId = body.contactId;
+
+			const visitor = await createVisitor(undefined, contact.name, email, phone);
+
+			const room = await createLivechatRoom(visitor.token);
```

Comment:
Is it possible to at least close the room after all?

Reply:
Done!